types/src/blockchain.rs

```diff
@@ -215,18 +253,47 @@ pub enum BlockchainResponse {
         cumulative_difficulty: u128,
     },
 
-    /// The response for [`BlockchainReadRequest::FindFirstUnknown`].
+    /// Response to [`BlockchainReadRequest::FindFirstUnknown`].
     ///
     /// Contains the index of the first unknown block and its expected height.
     ///
     /// This will be [`None`] if all blocks were known.
     FindFirstUnknown(Option<(usize, usize)>),
 
-    /// The response for [`BlockchainReadRequest::AltBlocksInChain`].
+    /// Response to [`BlockchainReadRequest::AltBlocksInChain`].
     ///
     /// Contains all the alt blocks in the alt-chain in chronological order.
     AltBlocksInChain(Vec<AltBlockInformation>),
 
+    /// Response to:
+    /// - [`BlockchainReadRequest::Block`].
+    /// - [`BlockchainReadRequest::BlockByHash`].
+    Block(Block),
+
+    /// Response to [`BlockchainReadRequest::TotalTxCount`].
+    TotalTxCount(usize),
+
+    /// Response to [`BlockchainReadRequest::DatabaseSize`].
+    DatabaseSize {
+        /// The size of the database file in bytes.
+        database_size: u64,
+        /// The amount of free bytes there are
+        /// the disk where the database is located.
+        free_space: u64,
+    },
+
+    /// Response to [`BlockchainReadRequest::Difficulty`].
+    Difficulty(u128),
+
+    /// Response to [`BlockchainReadRequest::OutputHistogram`].
+    OutputHistogram(Vec<OutputHistogramEntry>),
+
+    /// Response to [`BlockchainReadRequest::CoinbaseTxSum`].
+    CoinbaseTxSum(CoinbaseTxSum),
+
+    /// Response to [`BlockchainReadRequest::MinerData`].
+    MinerData(MinerData),
```

Comment:
### Signatures (blockchain response)
Added to the existing `BlockchainResponse`.